numpy/lib/tests/test_stride_tricks.py

```diff
@@ -324,7 +325,9 @@ def test_as_strided():
     assert_equal(a.dtype, a_view.dtype)
 
     # Make sure that the only type that could fail is properly handled
-    dt = np.dtype({'names': [''], 'formats': ['V4']})
+    with warnings.catch_warnings(record=True) as w:
+        warnings.filterwarnings('always', '', DeprecationWarning)
+        dt = np.dtype({'names': [''], 'formats': ['V4']})
```

Comment:
After the deprecation period is over this should be changed to
```
dt = np.dtype({'names': ['foo'], 'formats': ['V4']})
```

Reply:
or just removed since it is specifically testing for empty fields

Reply:
moved to deprecation PR